pkg/util/validation/validation.go

```diff
@@ -74,3 +74,37 @@ func ValidatePolicyFieldSelector(fieldSelector *policyv1alpha1.FieldSelector) er
 
 	return nil
 }
+
+// ValidateOverrideSpec tests if the overrideRules and (overriders or targetCluster) co-exist
+func ValidateOverrideSpec(overrideSpec *policyv1alpha1.OverrideSpec) error {
+	if overrideSpec == nil {
+		return nil
+	}
+
+	if overrideSpec.OverrideRules == nil {
+		return nil
+	}
+
+	if overrideSpec.TargetCluster != nil || !validateOverridesEmpty(overrideSpec.Overriders) {
+		return fmt.Errorf("overrideRules and (overriders or targetCluster) can't co-exist")
+	}
+
+	return nil
+}
+
+// validateOverridesEmpty tests if the overriders of override policy is empty
+func validateOverridesEmpty(overriders policyv1alpha1.Overriders) bool {
```

Comment:
```suggestion
func EmptyOverrides(overriders policyv1alpha1.Overriders) bool {
```
This function just checks if the `.spec.Overriders` is empty, no validation functionality.

Reply:
done